scripts/globals/matrimonialcoffer.lua

```diff
@@ -0,0 +1,54 @@
+-----------------------------------
+-- Matrimonial Coffer NPCs
+-----------------------------------
+require("scripts/globals/zone")
+require("scripts/globals/npc_util")
+
+dsp = dsp or {}
+dsp.matrimonialcoffer = dsp.matrimonialcoffer or {}
+
+function dsp.matrimonialcoffer.startEvent(player)
+    local race = player:getRace()
```

Comment:
Seems like `getGender` might be a better fit?

Reply:
awesome @zynjec  didn't know how to use that since only saw it being used for dancer but this is hella awesome. Appreciate it.